.golangci.yml

```diff
@@ -1,46 +1,28 @@
-# Options for analysis running.
 run:
-  # The default concurrency value is the number of available CPU.
-  # concurrency: 4
-  # Timeout for analysis, e.g. 30s, 5m.
-  # Default: 1m
   timeout: 10m
-  # Include test files or not.
-  # Default: true
-  tests: false
-  # Which dirs to skip: issues from them won't be reported.
-  # Can use regexp here: `generated.*`, regexp is applied on full path.
-  # Default value is empty list,
-  # but default dirs are skipped independently of this option's value (see skip-dirs-use-default).
-  # "/" will be replaced by current OS file path separator to properly work on Windows.
-  skip-dirs:
-    - runtime/sql
-    - proto/gen
-  # Which files to skip: they will be analyzed, but issues from them won't be reported.
-  # Default value is empty list,
-  # but there is no need to include all autogenerated files,
-  # we confidently recognize autogenerated files.
-  # If it's not please let us know.
-  # "/" will be replaced by current OS file path separator to properly work on Windows.
-  skip-files:
-    - ".*\\.my\\.go$"
-  issues-exit-code: 1
+  tests: false # TODO: Change to true
```

Comment:
Does this require addressing before merging ?

Reply:
No it was also false before and gives a lot of errors, so should be done separately. Actually I will just remove the comment and put it in the code quality burndown.